### PR TITLE
Update docs io-5571

### PR DESCRIPTION
**Pull request by Jben13**
image
image

https://ioffice.myjetbrains.com/youtrack/issue/io-5571

### CODE REVIEW

File: api/api.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ The base URL will be https://`siteaddress`/external/api/rest/v2/
 
 All endpoints in the document are relative to this url.
 
+Note: The term "rooms" has been amended to "spaces" any references to a room or rooms will now be referred to as space or spaces. This change does not affect the current endpoints.  
+
```
Comment (RedStarThrower) [inline api/api.md:13]: Note: The term "room" has been amended to "space"; all references to a "room" or "rooms" will now be referred to as a "space" or "spaces". This change does not affect current endpoints.

**Review by RedStarThrower — CHANGES_REQUESTED** [review]
Just some grammar updates

File: api/api.md (hunk 1 above)
Comment (rmoscowitz) [reply api/api.md:13]: @Jben13 after you make this change, don't forget to run `make export` and commit the new index.html file too

Comment (Jben13) [reply api/api.md:13]: @RedStarThrower @rmoscowitz 
Thanks for the input. I have update the changes to the note.

**Review by RedStarThrower — APPROVED** [review]
💯

**Pull request merged**